### DISCUSSION TITLE
"G-Helper is already running" popup waiting for me on every boot

### DISCUSSION THREAD

**Discussion by dino-halilovic** [seerge/g-helper#3417] [Q&A] [answered]
As title says, plus it says to check taskbar or something like that for the icon. I would guess it tries to start using 2 methods, so 2nd method becomes unneeded when the first one finishes. How would I fix this because its annoying to close the popup every boot...

**Comment by seerge** [comment, marked as answer]
@velikidizel hello, you seem to have scheduled app twice under different user accounts in windows.

Try to run it as admin and then uncheck and check “Run on Startup” checkbox.

**Reply by dino-halilovic** [reply]
I do not have 2 user accounts, nor made any changes related to accounts or this to cause. Your suggestion worked, just to recheck the Run on Startup. Thanks
